middlewares/authorization.js

```diff
@@ -0,0 +1,21 @@
+const {Project} = require('../models')
+function authorization(req,res,next){   
+    const id = req.params.id
+    Project.findOne({ 
+        where:{
+            TodoId: id,
+            UserId: req.currentUserId
+        }})
+
+    .then(result=>{        
+            return next()
+    })
+    .catch(err=>{
+        res.status(404).json({
+            message:"Error not found",
+            todos:err
```

Comment:
ini errornya maksudnya apa ya?
ini error authorisasi project kan ya?
todos itu buat apa?
message nya error not found?

Reply:
fixed